core/modules/node/node.path.inc

```diff
@@ -29,6 +30,7 @@ function node_path_info() {
   }
 
   foreach (node_type_get_types() as $node_type_name => $node_type) {
+    $info['node']['alias bulk actions items'][$node_type_name] = t('URL alias for <strong>%node_label</strong> <small>(Machine name: <code>!machine_name</code>)</small>', array('%node_label' => $node_type->name, '!machine_name' => $node_type->type));
```

Comment:
And "URL alias" should be plural since you're doing bulk updates on multiple aliases.